style.css

```diff
@@ -0,0 +1,3 @@
+form * {
+  display: block;
+}
```

Comment:
- [ ] Great job so far, it's one of the project requirements to use the style that matches the wireframe. Kindly add styling to your site.
- [ ] **Great job using Microverse's template for your readme but you didn't customize it to this particular project please do that.**

## Wireframe

![image](https://user-images.githubusercontent.com/59416136/165141884-e81c2de4-9ca1-433f-aab7-35a4a6a6ac0a.png)